gcloud/bigquery/connection.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Create / interact with gcloud bigquery connections."""
+
+from gcloud import connection as base_connection
+
+
+SCOPE = ('https://www.googleapis.com/auth/bigquery',
+         'https://www.googleapis.com/auth/cloud-platform')
```

Comment:
I'm unclear on what we should be doing in these situations. The `cloud-platform` scope encompasses like every service, so the `bigquery` scope is essentially useless when paired with `cloud-platform`.

/cc @jgeewax